api/lib/domain/models/CertificationCenterInvitation.js

```diff
@@ -10,9 +10,18 @@ const statuses = {
   CANCELLED: 'cancelled',
 };
 
+const roles = {
+  ADMIN: 'ADMIN',
+  MEMBER: 'MEMBER',
+  AUTO: null,
+};
+
 const validationScheme = Joi.object({
   id: Joi.number().optional(),
   email: Joi.string().email().optional(),
+  role: Joi.string()
+    .valid(...Object.values(roles))
+    .optional(),
```

Comment:
J'ai bien noté que la propriété `AUTO` avait été définie pour le code des _Organizations_ , mais à partir du moment où `role` est `optional` et où `AUTO` n'est utilisée nulle par ailleurs dans le code, il me semble inutile d'ajouter cette propriété et je suggère la simplification ci-dessous. 

```suggestion
const roles = {
  ADMIN: 'ADMIN',
  MEMBER: 'MEMBER',
};

const validationScheme = Joi.object({
  id: Joi.number().optional(),
  email: Joi.string().email().optional(),
  role: Joi.string()
    .valid(...Object.values(roles))
    .optional(),
```

Reply:
Je ne l'avais pas mise de base, mais j'ai finis par la rajouter car des tests cassaient.

Reply:
Ah, bien reçu ! Je n'avais pas lancé les tests avec cette modif ! Je vais lancer les tests avec cette modif pour me rendre compte !

Reply:
Correspond au commentaire sur la partie front plus bas 🙂